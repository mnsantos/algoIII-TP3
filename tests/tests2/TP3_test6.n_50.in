50 422
3 10
1 8
1 2
1 7
2 8
1 3
7 9
3 8
4 8
4 15
1 4
4 11
9 10
9 13
1 16
1 17
1 18
1 9
4 13
9 11
7 14
6 13
10 13
1 6
1 12
1 5
1 10
5 6
5 14
6 8
2 9
10 11
4 9
1 11
4 14
7 13
10 12
3 12
47 48
46 49
45 47
44 46
43 49
43 44
42 45
42 44
42 43
41 43
40 50
40 49
40 47
40 46
40 43
39 44
39 40
38 48
38 47
38 46
38 44
38 39
37 39
37 38
36 50
36 47
36 44
36 38
35 48
35 42
35 40
35 39
34 50
34 47
34 45
34 42
34 36
11 28
11 26
11 19
11 18
11 17
11 16
11 14
10 50
10 46
10 40
10 39
10 37
10 36
10 30
10 29
10 28
10 26
10 21
10 18
9 41
9 40
9 33
9 32
9 30
9 29
9 27
9 26
9 22
9 17
9 15
8 50
8 49
8 47
8 46
8 44
8 43
8 39
8 35
8 34
8 32
8 31
8 29
8 28
8 27
8 23
8 21
8 15
7 50
7 48
7 47
7 46
7 45
7 44
7 41
7 40
7 38
7 27
7 25
7 24
7 22
7 18
7 16
6 44
6 43
6 37
6 35
6 34
6 28
6 27
6 25
6 23
6 21
5 48
5 47
5 45
5 32
5 23
5 21
5 18
4 48
4 47
4 45
1 50
1 49
1 48
1 47
1 46
1 45
1 44
1 43
1 42
1 41
1 40
1 39
1 38
1 37
1 36
1 35
1 34
1 33
1 32
1 31
1 30
1 29
1 28
1 27
1 19
1 20
1 21
1 22
1 23
1 24
1 25
1 26
2 19
2 21
2 25
2 28
2 30
2 31
2 33
2 34
2 36
2 37
2 38
2 39
2 40
2 42
2 45
2 47
3 20
3 22
3 24
3 27
3 28
3 29
3 30
3 32
3 35
3 41
3 44
3 50
4 20
4 21
4 24
4 27
4 32
4 36
4 38
4 42
11 33
11 34
11 39
11 42
11 43
11 44
11 45
11 47
11 49
12 15
12 22
12 23
12 24
12 27
12 31
12 32
12 33
12 38
12 44
12 46
12 47
12 50
13 19
13 22
13 25
13 27
13 28
13 30
13 31
13 33
13 35
13 36
13 40
13 43
13 49
14 21
14 23
14 25
14 29
14 33
14 34
14 38
14 39
14 45
14 46
14 47
14 50
15 18
15 23
15 25
15 26
15 32
15 41
15 42
15 43
15 44
15 45
15 47
16 22
16 33
16 34
16 37
16 39
16 42
16 43
16 46
16 50
17 19
17 23
17 28
17 32
17 34
17 36
17 37
17 42
17 43
17 49
18 19
18 20
18 25
18 26
18 28
18 33
18 34
18 36
18 38
18 39
18 41
18 43
18 50
19 25
19 28
19 29
19 30
19 32
19 37
19 38
19 41
19 46
19 50
20 22
20 27
20 29
20 30
20 32
20 36
20 39
20 45
21 22
21 25
21 27
21 35
21 42
21 43
21 50
22 23
22 28
22 31
22 36
22 37
22 39
22 40
22 47
23 27
23 29
23 30
23 37
23 41
23 43
23 46
24 25
24 32
24 34
24 35
24 36
24 41
24 43
24 45
24 46
24 48
25 35
25 38
25 40
25 46
26 27
26 30
26 35
26 36
26 40
27 30
27 35
27 37
27 42
27 48
28 31
28 35
28 40
28 44
28 48
29 33
29 35
29 36
29 38
29 39
29 49
29 50
30 32
30 34
30 37
30 38
30 39
30 46
30 47
31 36
31 38
31 39
31 45
31 48
31 50
32 33
32 36
32 41
32 46
32 50
33 34
33 35
33 36
33 37
33 40
33 41
33 42
33 45
1 15
1 13
1 14
3 5
5 16
0
