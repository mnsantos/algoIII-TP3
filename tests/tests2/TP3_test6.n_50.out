110 6 1 7 9 13 22 40
